stroeer/core/v1/article.proto

```diff
@@ -8,133 +8,318 @@ option go_package = "github.com/stroeer/tapir/go/stroeer/core/v1;core";
+// only contain teaser relevant information (e.g. with no Body field).
+//
+// Message fields and map entries are optional unless commented otherwise.
 message Article {
+  // Unique ID of the article defined by the content management system (required).
   int64 id = 1;
+
+  // Main content type of the article.
   ContentType type = 2;
+
+  // Subtype of the article.
   ContentSubType sub_type = 3;
+
+  // References to get access to the article on different technical platforms like
+  // canonical web or AMP (required).
   References references = 4;
+
+  // Hierarchical section information of the article (required).
   Section section = 5;
+
+  // Generic map containing general content and configuration information of
+  // the article (required).
+  //
+  // The entry set is defined by the content management system and
+  // will vary depending on the main type of the article.
+  //
+  // For CONTENT_TYPE_ARTICLE, this map will always contain non empty values
+  // for the following keys:
+  //
+  // * headline
+  // * top_line
+  // * summary
+  // * meta_title
+  //
+  // For CONTENT_TYPE_GALLERY, this map will always contain non empty values
+  // for the following keys:
+  //
+  // * headline
+  //
+  // Clients must be resilient to unknown or missing entry sets.
   map<string, string> fields = 6;
+
+  // Recursive body structure of the article to render on detail
+  // pages (optional).
+  //
+  // This message will be empty for the teaser representation of the article.
+  //
+  // see https://github.com/orgs/stroeer/teams/api/discussions/1
   Body body = 7;
+
+  // Technical metadata of the article (optional).
   Metadata metadata = 8;
+
+  // Elements of the article to render as a teaser e.g. on overview pages (optional).
   repeated Element elements = 9;
+
+  // Entities of the article like persons, places etc. (optional).
   repeated string keywords = 10;
+
+  // Unique IDs of articles related to this article (optional).
+  //
+  // Related articles are defined manually in the content management system
+  // by the editorial department.
   repeated int64 onwards = 12;
 }
 
+// Technical metadata of the article.
 message Metadata {
+
+  // State of the article in the content management system (optional).
+  // see https://github.com/stroeer/tapir/issues/64
   string state = 1;
-  //  editorial date: Gültig von
+
+  // Manually set editorial date ('Gültig von') at which the article is valid to deliver on
+  // digital platforms in seconds of UTC time since Unix epoch (optional).
   google.protobuf.Timestamp valid_from = 2;
-  //  editorial date: Gültig bis
+
+  // Manually set editorial date ('Gültig bis') till the article is valid to deliver on
+  // digital platforms in seconds of UTC time since Unix epoch (optional).
   google.protobuf.Timestamp valid_to = 3;
-  //  editorial date: Zuletzt bearbeitet
+
+  // editorial date: Zuletzt bearbeitet
+  // TODO: this seems unused....remove?
```

Comment:
many of the date fields vanished from the CMS lately, this should be dealt with in a separate PR.

Reply:
see #112